File: Data Structures & Algorithms/max-consecutive-ones/submission-0.swift
class Solution {
    func findMaxConsecutiveOnes(_ nums: [Int]) -> Int {
        var count = 0
        var maxCount = 0

        for num in nums {
            count = num == 1 ? count + 1 : 0
            maxCount = max(maxCount, count)
        }

        return maxCount
    }
}

File: Data Structures & Algorithms/move-zeroes/submission-0.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        non0 = []
        for num in nums:
            if num != 0:
                non0.append(num)

        for i in range(len(nums)):
            if i < len(non0):
                nums[i] = non0[i]
            else:
                nums[i] = 0


        